addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -460,6 +455,7 @@ def put_discount_on(discount_line_vals, other_line_vals):
                 json_payload['ItemList'].remove(discount_line)
         if not discount_lines:
             return json_payload
+        invoice.message_post(body="Negative lines will be decreased from positive invoice lines having the same taxes and HSN code")
```

Comment:
```suggestion
        invoice.message_post(body=_("Negative lines will be decreased from positive invoice lines having the same taxes and HSN code"))
```
Is it clear to the user it is about sending to the government?